main.go

```diff
@@ -83,13 +86,19 @@ func main() {
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
+		Scheme: scheme,
```

Comment:
I did my best to make azure-private-endpoint-operator working again. when I changed zap Development flag it was not even building due to a lot of changes in controller-manager